docs/api/hardhat/migrating-to-zksync.md

```diff
@@ -1,22 +1,18 @@
 # Migration guide
 
-Do you have a Hardhat project and you want to deploy it to zkSync? In this page you'll find all the steps you need to do to migrate an existing Hardhat project to zkSync.
-
+This guide shows you how to migrate an existing hardhat project to zkSync Era.
 ## Overview
 
-zkSync offers [multiple Hardhat plugins](./plugins.md) with different features but in this guide we'll focus on only the ones you'd need to do to migrate your project to zkSync.
-
-:::warning Non-default paths are not supported yet.
-
-Contracts files must be included in the `contracts` folder and deployment scripts must be included in the `deploy` folder.
-
-Support for custom paths will be included in the future.
+zkSync Era offers [multiple hardhat plugins](./plugins.md) with different features but in this guide, we are focused on only the required few needed to migrate your project to zkSync Era.
 
+::: warning Non-default paths are not supported yet
+- Contract files must be included in the `contracts` folder and deployment scripts must be included in the `deploy` folder.
+- Support for custom paths will be included in the future.
```

Comment:
```suggestion
- Support for custom paths will be included in the future.
```
Is this still true?

Reply:
So it's not live yet, but we'll soon release support to configure the path for the deployment scripts. 
So we can keep the message